src/robot_features.cpp

```diff
@@ -111,6 +112,50 @@ FeatureNavigation::FeatureNavigation(const YAML::Node& nav_conf) : FeatureWithDr
   this->driver_enabled_ = true;
 }
 
+FeatureGripper::FeatureGripper() : FeatureWithDriver("gripper")
+{
+}
+
+FeatureGripper::FeatureGripper(const YAML::Node& grip_conf)
+  : FeatureWithDriver("gripper")
+{
+  this->package_name_ = grip_conf["controller"]["package_name"].as<std::string>();
+  if (grip_conf["controller"]["executable"])
+  {
+    this->executable_ = grip_conf["controller"]["executable"].as<std::string>();
+  }
+  else
+  {
+    this->executable_ = "move_group.launch";      // For now a defaul value.. 
```

Comment:
Why is the default value "move_group.launch" ? And from which package this launch file would come from?